main.tf

```diff
@@ -1,44 +1,64 @@
-resource "castai_eks_cluster" "my_castai_cluster" {
-  account_id                 = var.aws_account_id
-  region                     = var.aws_cluster_region
-  name                       = var.aws_cluster_name
-  subnets                    = var.subnets
-  override_security_groups   = var.override_security_groups
-  dns_cluster_ip             = var.dns_cluster_ip
-  ssh_public_key             = var.ssh_public_key
-  tags                       = var.tags
+resource "castai_eks_cluster" "this" {
```

Comment:
I like this rename but when updating terraform module user will remove and re-onboard the cluster. We should warn about it.

Reply:
I will double check it but I don't think it has impact on actual resources